pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -390,38 +440,12 @@ func (c *Controller) syncProvisionRequestsPodTemplates(ctx context.Context, wl *
 			return err
 		}
 
-		if err != nil {
-			// it's a not found, so create it
-			newPt := &corev1.PodTemplate{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      ptKey.Name,
-					Namespace: ptKey.Namespace,
-					Labels: map[string]string{
-						constants.ManagedByKueueLabel: "true",
-					},
-				},
-				Template: ps.Template,
-			}
-
-			// apply the admission node selectors to the Template
-			psi, err := podset.FromAssignment(ctx, c.client, psaMap[psName], reqPS.Count)
-			if err != nil {
-				return err
-			}
-
-			err = podset.Merge(&newPt.Template.ObjectMeta, &newPt.Template.Spec, psi)
-			if err != nil {
-				return err
-			}
-
-			// copy limits to requests if needed
-			workload.UseLimitsAsMissingRequestsInPod(&newPt.Template.Spec)
```

Comment:
Since we moved this part to different place, can we name the function accordingly?

Reply:
I believe the function name is correct because we still need to sync controllerReference in PodTemplate. Do you have a different name in mind?

Reply:
I think both functions `createPodTemplate` and `UseLimitsAsMissingRequestsInPod` are named ok. 

I'm not sure these names are best possible, but they don't strike me as misleading.